rfc/0054-sdk3-management-apis.md

```diff
@@ -3599,11 +3653,18 @@ interface CollectionSpec {
     String ScopeName();
 
     Duration MaxExpiry();
+    
+    Bool History();
 }
 ```
 
 `MaxExpiry` is the time in seconds for the TTL for new documents in the collection. Omit if not set.
 
+`History` is whether history retention override is enabled on this collection. Omit if not set - not set will default to bucket level setting.
+* On Create/Update this setting should be guarded by the bucket capability of `nonDedupedHistory`
+* If the bucket capability is not supported then a `FeatureNotAvailableException` must be raised.
+  * Suggested supplementary text: "history is unsupported by the server, are you using a bucket with the magma storage engine?"
```

Comment:
So no control over bytes & duration at the collection-level?

Is there any way to unset a true/false back to the default state, e.g. default to the bucket level setting?

Reply:
> So no control over bytes & duration at the collection-level?

Correct.

> Is there any way to unset a true/false back to the default state, e.g. default to the bucket level setting?

I'll check that and update the RFC either way.

Reply:
> Is there any way to unset a true/false back to the default state, e.g. default to the bucket level setting?

No, the setting is inherited from the bucket at creation time. Thereafter the setting is owned at the collection level. Omitting `history` at update time will just not alter the setting.

Reply:
Ok, thanks for checking